hyper/http20/stream.py

```diff
@@ -90,7 +90,7 @@ def __init__(self,
         # There are two flow control windows: one for data we're sending,
         # one for data being sent to us.
         self._in_window_manager = window_manager
-        self._out_flow_control_window = 65535
+        self._out_flow_control_window = 2**24
```

Comment:
Again, this *is* a flow control window, but not one we control: the remote side of the connection owns it.